run.py

```diff
@@ -0,0 +1,125 @@
+'''
+Full TPS workflow 
+end-point: Cytoscape session 
+
+'''
+
+
+import yaml
+import pprint
+import sys
+import subprocess
+import os
+import csv
+import time
+import pandas as pd
+from workflow.table_generation import PrepTemporalCytoscapeTPS
+from workflow.table_refactor import refactor_col_delim
+from workflow.visualization_utilities import vis, find_path, process_exists
+from workflow.parser import Parser
+from workflow.visualization import Visualization
+from requests.exceptions import ConnectionError as CE
+from requests.exceptions import HTTPError
+from json.decoder import JSONDecodeError
+
+
+
+
+def main(args):
+
+    CONFIG = args[1]
+    config = os.path.abspath(CONFIG)
+
+    with open(config) as c:
+        params = yaml.load(c, Loader = yaml.FullLoader)
+
+    parser = Parser(params)
+    sys.stdout.flush()
+    viz_engine  = Visualization(params)
+    sys.stdout.flush()
+    outputs, out_folder, label = parser.parse()
+    annot, style = viz_engine.generate_annotations(outputs, out_folder)
+    sys.stdout.flush()
+
+    print("OUTPUTS_LIST: ", outputs)
+
+
+    CYTOSCAPE = "Cytoscape.exe"
+    DIRNAME = params['Cytoscape']["path"]
+    OUTPUT_FILE = os.path.join(out_folder, label+"-output.sif")
+    outStyleFile = style
+    REFACTORED_ANNOTATIONS = annot
+    SAVE_FILE = os.path.join(out_folder, params['Cytoscape']["session"])
+
+    print("CYTOSCPAPE: ", CYTOSCAPE)
+    print("DIRNAME: ", DIRNAME)
+    print("OUTPUT: ", OUTPUT_FILE)
+    print("annotations: ", REFACTORED_ANNOTATIONS)
+    print("SAVE_FILE: ", SAVE_FILE)
+
+
+     # check if Cytoscape is running
+    is_running = process_exists('Cytoscape.exe')
+
+    if (is_running == False):
+        print("Cytoscape not running")
+
+        # find path to Cytoscape on machine
+        cyto_path = find_path(CYTOSCAPE, DIRNAME)
```

Comment:
`find_path` assumes that there is a successful match.  When I ran with `config.yaml`, which has an invalid path to Cytoscape for my machine, I got this error:
```
Cytoscape not running
Traceback (most recent call last):
  File "run.py", line 124, in <module>
    main(sys.argv)
  File "run.py", line 68, in main
    cyto_path = find_path(CYTOSCAPE, DIRNAME)
  File "<my-path>\tps\workflow\visualization_utilities.py", line 64, in find_path
    return result[0]
IndexError: list index out of range
```